src/test/js/shared/idam_helper.js

```diff
@@ -254,14 +244,14 @@ class IdamHelper extends Helper {
     }
 
     createUserWithRoles(email, forename, userRoles) {
-        var codeUserRoles = [];
-        for (var i = 0; i < userRoles.length; i++) {
+        let codeUserRoles = [];
+        for (let i = 0; i < userRoles.length; i++) {
             codeUserRoles.push({'code': userRoles[i]});
         }
```

Comment:
minor: this for loop could be rewritten as:
`const codeUserRoles = userRoles.map(role -> ({ code: role });`

Reply:
Done